jest-setup.ts

```diff
@@ -0,0 +1,3 @@
+jest.mock("react-json-tree", () => ({
+  JSONTree: () => '<div data-testid="mock-json-tree" />',
+}));
```

Comment:
@PacoVK that is weird, I've needed to mock the JSONTree library to workarround this issue

`/node_modules/react-json-tree/lib/index.js:5
    import React, { useMemo } from 'react';
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module`
   
Other workarround could be using babel for transform this package to a module systanxis, but I haven't luck implementing it